php_opcache/README.md

```diff
@@ -0,0 +1,85 @@
+# Agent Check: php_opcache
+
+## Overview
+
+This check monitors [PHP OPcache][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the `php_opcache` check on your host:
+
+
+1. Install the [developer toolkit][3].
+ on any machine.
+
+2. Run `ddev release build php_opcache` to build the package.
+
+3. [Download the Datadog Agent][4].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/php_opcache/dist/<ARTIFACT_NAME>.whl`.
+
+#### OPcache Setup
+
+
+OPcache does not expose metrics by default so this integration includes a metric exporter, located here:
+
+    ```
+    /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_opcache/assets/exporter/opcache-dd-handler.php
+    ```
+
+When you configure your Agent (the `instances` setting, described next), you can refer to the exporter directly by this file name, or you can configure an alias for it on your web server. For example, if you're using Apache, the alias in the web server configuration file would look like this:
```

Comment:
I didn't want to assume that it would be retrieved from a URL, I would prefer to simply install it and refer to it.

It's difficult to write documentation for containers and other complex situations, and I expect advanced users to figure it out on their own.

It's also a good idea to include the URL.
However, I haven't decided where to put the .php yet, so I'll do it later.

Reply:
Yeah I completely agree, having docs for every edge case wouldn't be reasonable. My comment is just about providing a direct link to the .php file for advanced users.

Reply:
80bc5afc38c2efdb7ec578bc77db5794ed780a06